bazel/variables.bzl

```diff
@@ -1,4 +1,37 @@
-DEFAULT_COPTS = [
+# Those are the build flags used to build MalDocA (maldoca/*).
+# The flags are not applied to build third party dependencies.
```

Comment:
What do you mean by "The flags are not applied to build third party dependencies."?

Reply:
those flags r only applied to build maldoca/* not maldoca's third party dependencies (i.e. stuff in third_party/* (e.g. chromium)). updated the comment.